### ISSUE TITLE
Adjust automatic colors to meet a11y standards 

### ISSUE THREAD

**Issue by BANOnotIT** [readrops/Readrops#172]
In 1.3.1 the feed name is using color from the icon/image. But that color is not adjusted to match background especially in dark themes. Thus failing checks, like WCAG contrast check. 

For example https://esif.dev with dark theme [fails WCAG's AA](https://webaim.org/resources/contrastchecker/?fcolor=074AD2&bcolor=454545).

**Comment by Shinokuni** [comment]
There is check for the extracted color to be not too dark or too bright, but maybe the threshold is not enough suited to pass a contrast check.

**Comment by Shinokuni** [comment]
So this should be fixed in 9cdb3f47. Colors will now be displayed more dynamically depending on the contrast with the background. The contrast threshold has been also raised. Lastly the next release will contain a way to change feeds color.
